lib/internal/Magento/Framework/Pricing/PriceCurrencyInterface.php

```diff
@@ -78,11 +78,21 @@ public function convertAndFormat(
     /**
      * Round price
      *
+     * @deprecated
      * @param float $price
      * @return float
      */
     public function round($price);
 
+    /**
+     * Round price with precision
+     *
+     * @param float $price
+     * @param int $precision
+     * @return float
+     */
+    public function roundPrice($price, $precision = self::DEFAULT_PRECISION);
```

Comment:
I don't think it's safe to add this method to an Interface, as it will break the classes implementing the interface but missing this method. Let's keep it only in the PriceCurrency class.

Reply:
Removed this from the interface class to be backward compatible. Eventually we want to make sure we have this function in the interface and delete the old function from the interface. Since there is not a 2.4-develop branch yet. How are we making sure this will be done in the next release of Magento?

Reply:
@sivaschenko Please have a look at my question above. How do we make sure that this will be changed in the 2.4 branch?

Reply:
Hi @dverkade if you'd like to remove the method it should be maked with deprecated tag, additionally you can create an issue describing what needs to be done, and finally create a pull request to 2.4-develop when the branch will be available.